docs/topics/i18n/translation.txt

```diff
@@ -1079,9 +1079,9 @@ interface within your Python code::
 The ``ngettext`` function provides an interface to pluralize words and
 phrases::
 
-    var object_count = 1 // or 0, or 2, or 3, ...
+    const objectCount = 1 // or 0, or 2, or 3, ...
     s = ngettext('literal for the singular case',
```

Comment:
Should `s` also be declared? I realized it was not before, so I don't know if it is deliberate or not.

Reply:
Good point, I think it should be.